src/contrib/cluster/Akka.Cluster.Tools/Singleton/ClusterSingletonProxySettings.cs

```diff
@@ -99,8 +99,8 @@ public ClusterSingletonProxySettings(string singletonName, string role, TimeSpan
                 throw new ArgumentNullException(nameof(singletonName));
             if (singletonIdentificationInterval == TimeSpan.Zero)
                 throw new ArgumentException("singletonIdentificationInterval must be positive", nameof(singletonIdentificationInterval));
-            if (bufferSize <= 0)
-                throw new ArgumentException("bufferSize must be positive", nameof(bufferSize));
+            if (bufferSize < 0)
+                throw new ArgumentException("bufferSize must not be negative", nameof(bufferSize));
```

Comment:
The `0` value is actually supported in the code: https://github.com/akkadotnet/akka.net/blob/b76b98af84232b45327cc8d0a336d39434d9018c/src/contrib/cluster/Akka.Cluster.Tools/Singleton/ClusterSingletonProxy.cs#L252

So this is just a minor validation issue.